### ISSUE TITLE
Dead link to gif in Part 1.4 (animation to find font name)

### ISSUE THREAD

**Issue by james-lyon** [nobiot/Zero-to-Emacs-and-Org-roam#13]
Hi, the link to the gif in Part 1.4 at the end of the "Set your font" section no longer exists (https://github.com/nobiot/Zero-to-Emacs-and-Org-roam/blob/main/images/2020-06-16_18-16-01.gif)

**Comment by nobiot** [comment]
Yes... I am in the process of putting new gif but haven't got around to it yet. Thank you for this note; it's good to know someone cares about them ;)

**Comment by james-lyon** [comment]
Thank you for the guide, I have just successfully followed it up to the end of Part 1 and everything is working correctly 😄

**Comment by nobiot** [comment]
Glad that it's useful to you :)
